packages/transaction-controller/src/index.ts

```diff
@@ -1,5 +1,61 @@
-export * from './TransactionController';
+export type {
```

Comment:
Making these exports explicit makes it easier to see changes to the public API. There should be no removals aside from those noted in the changelog.